node/Makefile

```diff
@@ -0,0 +1,2 @@
+test: 
+	go test -v -ldflags '-extldflags "-Wl,--allow-multiple-definition" ' ./...
```

Comment:
Please configure your editor to add trailing newlines. This is annoying to look at in the terminal.

Reply:
fixed.